clients/cobol-lsp-vscode-extension/src/__tests__/services/copybook/download/CopybookDownloaderForDsnTest.spec.ts

```diff
@@ -132,7 +132,8 @@ describe("Tests Copybook download from DNS", () => {
         "document-uri",
         "DNS.PATH",
       );
-      expect(allMemberMock).toHaveBeenCalledWith("DNS.PATH");
+      // cache resolves the members
+      expect(allMemberMock).not.toHaveBeenCalled();
```

Comment:
I'd suggest to make a new test (for datasets and uss), one verifies that the api is called, the other verifies usage of the cache.

Reply:
updated